modules/client-configuration/pages/clients-opensuseleapmicro.adoc

```diff
@@ -63,10 +60,10 @@ ifeval::[{uyuni-content} == true]
 .{opensuse} Channels - CLI
 |===
 
-| OS Version          | {leapmicro} 5.3
-| Base Channel	      | opensuse_micro5_3
-| Client Channel      | opensuse_micro5_3-sle-updates
-| SLE Updates Channel | opensuse_micro5_3-uyuni-client
+| OS Version          | {leapmicro} 5.x
+| Base Channel	      | opensuse_micro5_x
+| Client Channel      | opensuse_micro5_x-sle-updates
+| SLE Updates Channel | opensuse_micro5_x-uyuni-client
```

Comment:
In case you actually want to stay with "x" here, it would be good if you could add a hint that the user must replace it and where to find the real numbers.

Reply:
This entire table needs to be rewritten, which involves a bit more work. I'll do that in a separate PR.